src/utils/generate_cxx_code.cpp

```diff
@@ -254,7 +423,7 @@ const std::string  wcs::generate_cxx_code::generate_code(const LIBSBML_CPP_NAMES
   using  constant_init_ass = std::unordered_map<std::string, const ASTNode *>;
   typename constant_init_ass::const_iterator cinitasit;
   //  A map for constants in initial assignments
-  constant_init_ass sconstant_init_assig;
+  constant_init_ass sconstant_init_assig, sconstant_init_assig_notused;
 
   using  rate_rules = std::unordered_map <std::string, const ASTNode *>;
```

Comment:
model_constants -> model_constants_t
initial_assignments -> initial_assignments_t
constant_init_ass -> constant_init_assgn_t
rate_rules -> rate_rules_t

Reply:
assignment_rules -> assignment_rules_t
model_reactions -> model_reactions_t
wcs_struct_map_val -> wcs_struct_map_val_t
wcs_struct_map -> wcs_struct_map_t
event_assignments -> event_assignments_t
reaction_parameters -> reaction_parameters_t
See if there are more.
Also break long lines such as 670, 804, 863, 994 and so on that goes beyond 80+

Reply:
Done